pegasus/sites.v3/code.org/public/educate/curriculum/elementary-school.md

```diff
@@ -7,6 +7,8 @@ style_min: true
 
 <link href="/shared/css/course-blocks.css", type="text/css", rel="stylesheet"></link>
 
+<div style="clear:both; height: 100px;">&nbsp;</div>
```

Comment:
Currently, I add in some breathing room to each page that has a banner. Ideally, the banner would have that space built in. @breville clever tips on how to do this? I think I'm getting some funny interactions with clears, etc